src/awsexplorer/awsExplorer.ts

```diff
@@ -43,6 +43,12 @@ export class AwsExplorer implements vscode.TreeDataProvider<AWSTreeNodeBase>, Re
         this._onDidChangeTreeData = new vscode.EventEmitter<AWSTreeNodeBase | undefined>()
         this.onDidChangeTreeData = this._onDidChangeTreeData.event
         this.regionNodes = new Map<string, RegionNode>()
+        this.awsContext.onDidChangeContext((e) => {
```

Comment:
I missed this before, the returned event disposable needs to be properly cleaned up (same with line 53 if you can help fix existing code)